website/views/ref.slim

```diff
@@ -1,47 +1,75 @@
-/GENERATED BY rake gen:ref
-h3
-  | lazy
-  span.constraints title='should include Rsec::Helper' helper
-pre.desc
-  | Lazy parser is constructed when parsing starts. It is useful to reference a parser not defined yet
-i.example example:
-pre.code
-  code.prettyprint.lang-rb
+/GENERATED BY rake gen:ref

```

Comment:
This file is in fact generated from comments in code.
But since there are no more recent updates in the API, I guess it should be OK to do it...